### PR TITLE
Add LED matrix and 2 layouts to hub20

### PR DESCRIPTION
**Pull request by peepeetee**
## Types of Changes

Adds 2 layouts to hub20
Changed rgb implementation from rgb light to a rgb matrix implementation.

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by peepeetee** [comment]
@joshajohnson

**Comment by peepeetee** [comment]
my rgb_matrix_map_row_column_to_led_kb implementation does not work, help

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
> my rgb_matrix_map_row_column_to_led_kb implementation does not work, help

Your what?

**Comment by peepeetee** [comment]
> 
> 
> > my rgb_matrix_map_row_column_to_led_kb implementation does not work, help
> 
> Your what?

rgb_matrix_map_row_column_to_led_kb
see keyboards/hub20/hub20.c :124

There's 3 keys on the board with 2 LEDs lighting them, and I want them to both light up in the LED effects. Checkout the messages linked below

see https://discord.com/channels/440868230475677696/473506116718952450/829572669115727903
and https://discord.com/channels/440868230475677696/473506116718952450/827648566168256572

I just commented my implementation out for now. The function is not documented, I'll probably work on documenting it when(or if) I figure it out

**Comment by peepeetee** [comment]
What changes do I need to add?

File: keyboards/hub20/config.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 joshajohnson
+Copyright 2020 joshajohnson 2021 peepeetee
```
Comment (drashna) [inline keyboards/hub20/config.h:2]:
```suggestion
Copyright 2020 joshajohnson 
Copyright 2021 peepeetee
```

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**